client/src/components/ProfileSettings/ProfileDetails/ProfileDetails.tsx

```diff
@@ -1,91 +1,123 @@
-import { Avatar, Box, Button, Card, CardActionArea, CardContent, CardMedia, Grid, Paper, TextField, Typography } from "@material-ui/core";
+import {
+  Avatar,
+  Box,
+  Button,
+  Card,
+  CardActionArea,
+  CardContent,
+  CardMedia,
+  Grid,
+  Paper,
+  TextField,
+  Typography,
+} from '@material-ui/core';
 import Rating from '@material-ui/lab/Rating';
 import LocationOnIcon from '@material-ui/icons/LocationOn';
 import useStyles from './useStyles';
-import NavBar from '../../NavBar/AuthNavBar/AuthNavBar';
 import Navbar from '../../Navbar/Navbar';
 import profCover from '../../../Images/profcover.jpg';
 import profimg from '../../../Images/profimg.jpg';
 import doggo from '../../../Images/dog1.jpg';
 import doggo1 from '../../../Images/dog2.jpg';
 
 const ProfileDetails = (): JSX.Element => {
+  const classes = useStyles();
 
-    const classes = useStyles();
+  return (
+    <Grid container className={classes.root}>
```

Comment:
Some of this code seems like it's from a previous, please adjust the base of the PR to only reflect new changes.

Reply:
hii Rajiv, I think I branched it off main, not sure why this changes are visible in my commit, plus I don't know which branch should I rebase to ...

Reply:
If you would've branched off of main, these changes would not be reflecting as new changes. My guess is that you may have branched off of the profile details FE branch.